controllers/errors.go

```diff
@@ -0,0 +1,18 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/FideTech/yaus/config"
+)
+
+//ErrorHandler handles the hard coded "/e/" short links
+func ErrorHandler(w http.ResponseWriter, r *http.Request, value string) {
```

Comment:
For sanity I wonder if should call it errorlinkhandler since errorhsndlers sounds like handling errors

Reply:
Great point! I will change that